js/__fixtures__/contexts/credentials-v2-unstable.json

```diff
@@ -0,0 +1,289 @@
+          "@id": "cred:credentialSchema",
+          "@type": "@id",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "cred": "https://www.w3.org/2018/credentials#",
+            "JsonSchemaValidator2018": "cred:JsonSchemaValidator2018"
+          }
+        },
+        "credentialStatus": {
+          "@id": "cred:credentialStatus",
+          "@type": "@id"
+        },
+        "credentialSubject": {
+          "@id": "cred:credentialSubject",
+          "@type": "@id"
+        },
+        "evidence": {
+          "@id": "cred:evidence",
+          "@type": "@id"
+        },
+        "expirationDate": {
+          "@id": "cred:expirationDate",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "holder": {
+          "@id": "cred:holder",
+          "@type": "@id"
+        },
+        "issued": {
+          "@id": "cred:issued",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "issuer": {
+          "@id": "cred:issuer",
+          "@type": "@id"
+        },
+        "issuanceDate": {
+          "@id": "cred:issuanceDate",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "proof": {
+          "@id": "https://w3id.org/security#proof",
+          "@type": "@id",
+          "@container": "@graph"
+        },
+        "refreshService": {
+          "@id": "cred:refreshService",
+          "@type": "@id",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "cred": "https://www.w3.org/2018/credentials#",
+            "ManualRefreshService2018": "cred:ManualRefreshService2018"
+          }
+        },
+        "termsOfUse": {
+          "@id": "cred:termsOfUse",
+          "@type": "@id"
+        },
+        "validFrom": {
+          "@id": "cred:validFrom",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "validUntil": {
+          "@id": "cred:validUntil",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        }
+      }
+    },
+    "VerifiablePresentation": {
+      "@id": "https://www.w3.org/2018/credentials#VerifiablePresentation",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "cred": "https://www.w3.org/2018/credentials#",
+        
+        "holder": {
+          "@id": "cred:holder",
+          "@type": "@id"
+        },
+        "proof": {
+          "@id": "https://w3id.org/security#proof",
+          "@type": "@id",
+          "@container": "@graph"
+        },
+        "verifiableCredential": {
+          "@id": "cred:verifiableCredential",
+          "@type": "@id",
+          "@container": "@graph"
+        }
+      }
+    },
+    "JsonWebSignature2020": {
```

Comment:
We shouldn't be defining suites in the vc context in my opinion. Rather this should link to the external security v3 context. Otherwise we're going to always have to keep these two in sync which is much harder with the VC context needing more bureaucratic paperwork to update compared to the security-vocab.

Reply:
the current vc context defines suites: https://github.com/w3c/vc-data-model/blob/master/contexts/credentials/v1#L163

I think we need to do the following:

1. define suites in their own context, for example bbs-v1.json
2. define suites in the security context, for example security/v3-unstable.json
3. define suites in the vc context, for example credentials/v2

yes, keeping things in sync will suck without tests... I propose we write tests.

I don't know if the vc data model would accept an external context like sec/v3... IIRC that is the reason things look the way they do today, people wanted a single file, with no external contexts.

Reply:
In my opinion, I'd prefer to see us drop these terms from the V2-unstable context and start heading in the direction where only definitions related to that vocab / spec are defined in that context. I'd live with option 1 as well, but I think that goes against what the JSON only folks were after, so we'd probably need to make sure it doesn't present any problems for them as well, but that's got to be discussed in the VC maintenance WG.